CONTRIBUTING.md

```diff
@@ -1,358 +1,85 @@
-  nice people way better than mean ones!
-
-* Encourage diversity and participation: Make everyone in our community feel
-  welcome, regardless of their background and the extent of their
-  contributions, and do everything possible to encourage participation in
-  our community.
-
-* Keep it legal: Basically, don't get us in trouble. Share only content that
-  you own, do not share private or sensitive information, and don't break
-  the law.
-
-* Stay on topic: Make sure that you are posting to the correct channel and
-  avoid off-topic discussions. Remember when you update an issue or respond
-  to an email you are potentially sending to a large number of people. Please
-  consider this before you update. Also remember that nobody likes spam.
-
-* Don't send email to the maintainers: There's no need to send email to the
-  maintainers to ask them to investigate an issue or to take a look at a
-  pull request. Instead of sending an email, GitHub mentions should be
-  used to ping maintainers to review a pull request, a proposal or an
-  issue.
-
-### Guideline violations — 3 strikes method
-
-The point of this section is not to find opportunities to punish people, but we
-do need a fair way to deal with people who are making our community suck.
-
-1. First occurrence: We'll give you a friendly, but public reminder that the
-   behavior is inappropriate according to our guidelines.
-
-2. Second occurrence: We will send you a private message with a warning that
-   any additional violations will result in removal from the community.
-
-3. Third occurrence: Depending on the violation, we may need to delete or ban
-   your account.
+We currently use [dep](https://github.com/golang/dep) for dependency management.
 
-**Notes:**
+[good-first-issue]: https://github.com/cnabio/cnab-to-oci/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
+[help-wanted]: https://github.com/cnabio/cnab-to-oci/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
 
-* Obvious spammers are banned on first occurrence. If we don't do this, we'll
-  have spam all over the place.
+# Cutting a Release
 
-* Violations are forgiven after 6 months of good behavior, and we won't hold a
-  grudge.
+When you are asked to cut a new release, here is the process:
 
-* People who commit minor infractions will get some education, rather than
-  hammering them in the 3 strikes process.
+1. Figure out the correct version number, we follow [semver](semver.org) and
+   have a funny [release naming scheme][release-name]:
+   - Bump the major segment if there are any breaking changes.
+   - Bump the minor segment if there are new features only.
+   - Bump the patch segment if there are bug fixes only.
+   - Bump the build segment (version-prerelease.BUILDTAG+releasename) if you only
+     fixed something in the build, but the final binaries are the same.
+1. Figure out if the release name (version-prerelease.buildtag+RELEASENAME) should
```

Comment:
The release name was something that we were doing with `+ralpha` that we do anymore. We can probably drop that and simplify this (and seems like since this was copied, I need to update it in the other repos). 

So that would remove this li entry and modify the bullet point above to be 

```
- Bump the build segment (version-prerelease.BUILDTAG) if you only
     fixed something in the build, but the final binaries are the same.
```